plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/core-profiler.spec.js

```diff
@@ -43,8 +43,8 @@ test.describe( 'Store owner can complete the core profiler', () => {
 				} )
 			).toBeVisible();
 			await page
-				.getByRole( 'radio', { name: 'I’m just starting my business' } )
-				.first()
+				.getByRole( 'radio' )
+				.filter( { hasText: 'just starting my business' } )
```

Comment:
I noticed that tests are failing against WPCOM and Pressable for all strings that contains `’` character. Instead, I used `hasText` and partial text in all these cases.